src/actions/receive-profile.js

```diff
@@ -184,6 +185,7 @@ export function finalizeProfileView(
     // encoded into the URL.
     const selectedThreadIndexes = getSelectedThreadIndexesOrNull(getState());
     const pages = profile.pages;
+    const activeTabID = getActiveTabID(getState());
```

Comment:
super nit: this is used only in the block for `case 'active-tab':`, can you please define it there? That will make it easier to read when we get it closer to its use.
Note you'll need to put brackets around the case block so that eslint doesn't yell at you.